services/slam/client_test.go

```diff
@@ -268,6 +294,11 @@ func TestFailingClient(t *testing.T) {
 		test.That(t, err.Error(), test.ShouldContainSubstring, "failure during get internal state")
 		test.That(t, fullBytesInternalState, test.ShouldBeNil)
 
+		// test get latest map info
+		timestamp, err := failingSLAMClient.GetLatestMapInfo(context.Background())
+		test.That(t, err.Error(), test.ShouldContainSubstring, "failure to get latest map info")
+		test.That(t, timestamp == time.Time{}, test.ShouldBeTrue)
```

Comment:
[q] does ShouldResemble fail here?

Reply:
oh lol I didn't even think about that. Should I do that instead?